scripts/zones/Arrapago_Reef/mobs/Medusa.lua

```diff
@@ -6,11 +6,19 @@
 -----------------------------------
 package.loaded["scripts/zones/Arrapago_Reef/TextIDs"] = nil;
 -----------------------------------
-mixins = {require("scripts/mixins/job_special")};
+require("scripts/globals/status");
 require("scripts/zones/Arrapago_Reef/TextIDs");
+require("scripts/zones/Arrapago_Reef/MobIDs");
 require("scripts/globals/titles");
+mixins = {require("scripts/mixins/job_special")};
 -----------------------------------
 
+function onMobInitialize(mob)
+    mob:setMod(MOD_DEF, 400);
```

Comment:
skeptical, I'd like to see some kind of source data for where these values come from

Reply:
yeah, project only wants retail accurate, not the custom stuff.